### PR TITLE
Improvements to function material

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

File: exercises/Functions-nested-functions-R.md [hunk 1]
```diff
@@ -7,19 +7,12 @@ language: R
...
-(there are 2.205 pounds in a kilogram). Use that function along with your
-dinosaur mass function to estimate the weight, in pounds, of a 12 m long
-Stegosaurus (*12 m is about as big as they come and we want grandma's story to
-be as wild as possible*). In *Stegosauria*, `a` has been estimated as `10.95`
-and `b` has been estimated as `2.64`
+Measuring things using the metric system is the standard approach for
+scientists, but when communicating with your results more broadly it may be
+useful to use different units (at least in some countries). Write a function
+that converts kilograms into pounds (there are 2.205 pounds in a kilogram). Use
+that function along with your dinosaur mass function to estimate the weight, in
```
Comment (KristinaRiemer) [inline exercises/Functions-nested-functions-R.md]: Maybe change this to "along with your dinosaur mass function from Exercise 1...". They seem to struggle with knowing when an exercise is a followup to a previous exercise.

File: materials/functions-R.md [hunk 2]
```diff
@@ -42,18 +42,18 @@ function_name <- function(inputs) {
...
 ```
 
 * Creating a function doesn't run it.
 * Call the function with some arguments.
 
 ```
-add(2, 3)
-summed <- add(2, 3)
+calc_shrub_vol(0.8, 1.6, 2.0)
+shrub_vol <- calc_shrub_volume(0.8, 1.6, 2.0)
```
Comment (KristinaRiemer) [inline materials/functions-R.md]: Typo in function name, should be `calc_shrub_vol`

File: materials/functions-R.md [hunk 3]
```diff
@@ -65,56 +65,65 @@ summed <- add(2, 3)
...
-make_pie_filling <- function(stuff) {
-  filling <- pi * stuff
-  return(filling)
+est_shrub_mass <- function(volume){
+  mass <- 2.65 * volume^0.9
 }
+
+shrub_mass <- est_shrub_mass(calc_shrub_vol(0.8, 1.6, 2.0))
+
+shrub_mass <- calc_shrub_vol(0.8, 1.6, 2.0) %>%
```
Comment (KristinaRiemer) [inline materials/functions-R.md]: Will have to read in dplyr, so I'd probably add `library(dplyr)` the line before as a reminder.

File: materials/functions-R.md [hunk 4]
```diff
@@ -65,56 +65,65 @@ summed <- add(2, 3)
...
 ```
 
+* Calling functions from inside other functions
+* Allows organizing function calls into logical groups
+
 ```
-make_pie <- function(stuff, crust) {
-  pie <- crust + make_pie_filling(stuff)
-  return(pie)
+est_shrub_mass_dim <- function(length, width, height){
```
Comment (KristinaRiemer) [inline materials/functions-R.md]: Do you actually plan on demonstrating use of this function? If so, I'd add `est_shrub_mass_dim(0.8, 1.6, 2.0)` to the line after the creation of the function.

**Comment by ethanwhite** [comment]
Great suggestions. I implemented all of them.

**Comment by KristinaRiemer** [comment]
@ethanwhite does "dinosaur mass function" not refer to the allometric equation used in Exercise 1 ("Use and Modify")?

**Comment by ethanwhite** [comment]
> @ethanwhite does "dinosaur mass function" not refer to the allometric equation used in Exercise 1 ("Use and Modify")?

You're totally right. Fixed.

**Pull request merged**